fr.dwightstudio.JArmEmu.metainfo.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~            ____           _       __    __     _____ __            ___
+  ~           / __ \_      __(_)___ _/ /_  / /_   / ___// /___  ______/ (_)___
+  ~          / / / / | /| / / / __ `/ __ \/ __/   \__ \/ __/ / / / __  / / __ \
+  ~         / /_/ /| |/ |/ / / /_/ / / / / /_    ___/ / /_/ /_/ / /_/ / / /_/ /
+  ~        /_____/ |__/|__/_/\__, /_/ /_/\__/   /____/\__/\__,_/\__,_/_/\____/
+  ~                         /____/
+  ~     Copyright (C) 2023 Dwight Studio
+  ~
+  ~     This program is free software: you can redistribute it and/or modify
+  ~     it under the terms of the GNU General Public License as published by
+  ~     the Free Software Foundation, either version 3 of the License, or
+  ~     (at your option) any later version.
+  ~
+  ~     This program is distributed in the hope that it will be useful,
+  ~     but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~     GNU General Public License for more details.
+  ~
+  ~     You should have received a copy of the GNU General Public License
```

Comment:
What's the purpose of this header? You licensed the metadata by CC-1.0

Reply:
1. this is incosistent with the metadata license
2. this is not acceptable as metadata license

Reply:
Correct, it has been automatically generated by my IDE. I remove it